.codeclimate.yml

```diff
@@ -0,0 +1,30 @@
+engines:
+  brakeman:
+    enabled: true
+  bundler-audit:
+    enabled: true
+  csslint:
+    enabled: true
+  coffeelint:
+    enabled: true
+    config: .coffeelint.json
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+      - javascript
+  rubocop:
```

Comment:
I'm scared of rubocop until we re-do Matt's #451. But it's probably fine for now.